src/iso.rs

```diff
@@ -71,6 +72,123 @@ pub fn iso_embed(config: &IsoEmbedConfig) -> Result<()> {
     Ok(())
 }
 
+pub fn iso_embed_kargs(config: &IsoEmbedKargsConfig) -> Result<()> {
+    let mut holder = CopiedFileHolder::new(&config.input, config.output.as_ref())?;
+
+    // make sure we include trailing newline in calculations below
+    let mut inject_string = config.kargs.clone();
```

Comment:
Minor but we could go `mut-less` like:
```
let inject_string = if !inject_string.ends_with('\n') {
  let c = inject_string.clone();
  c.push('\n');
  Cow::Owned(c)
} else {
  Cow::Borrowed(inject_string.as_str())
}
```
or so.

(I know this is pointless here, I just try to avoid having `mut` bindings over long stretches of code unless necessary, and `Cow` :cow2: is cool)

Reply:
Thanks, tweaked! I usually try to avoid `mut` bindings too.